lib/components/app/call-taker-panel.js

```diff
@@ -147,12 +147,11 @@ class CallTakerPanel extends Component {
     const { expandAdvanced, planTripClicked } = this.state
     const advancedSearchStyle = {
       background: 'white',
+      boxShadow: 'none',
```

Comment:
Maybe move all of these styles to `.otp .advanced-search-options` in `calltaker.css`, instead, except for `display`. Also, `background`, `left`, and `right` can probably be deleted.